server/protocols/webadmin/webadmin-mailbox-deleted-message-vault/src/main/java/org/apache/james/webadmin/vault/routes/DeletedMessagesVaultRoutes.java

```diff
@@ -137,10 +184,12 @@ public String getBasePath() {
     @Override
     public void define(Service service) {
         service.post(RESTORE_PATH, this::userActions, jsonTransformer);
+        service.delete(ROOT_PATH, this::deleteWithScope, jsonTransformer);
+        service.delete(DELETE_PATH, this::deleteMessage, jsonTransformer);
     }
 
     @POST
-    @Path(ROOT_PATH)
+    @Path(USER_PATH)
```

Comment:
In the define method, the path is defined as RESTORE_PATH. Is there an inconsistency ?

Reply:
fixed. Please check it!